kong/db/dao/plugins/external.lua

```diff
@@ -0,0 +1,642 @@
+local cjson = require "cjson.safe"
```

Comment:
I think this file should be outside of `kong/db` and its filename should contain the word `server` on it. I suggest `kong/runloop/plugin_servers/init.lua`

Reply:
done